ubuntudesign/www.ubuntu.com
# Maas update

## Done

A collection of all page changes related to MAAS
 - /download/server
 - /download/server/provisioning
 - /server/maas

## QA

Visually QA the following pages in all viewports and check their respective copydocs
 - /download/server [copy](https://docs.google.com/document/d/1FFQQUtzFGH88hA30qnZm-3pwVqcVl9UYyrNT4qQWrSM/edit)
 - /download/server/provisioning [copy](https://docs.google.com/document/d/12GXYcP9WrOgoso6Sf3bQql6nALRUzpdlcRW3Gbu-npM/edit#)
 - /server/maas [copy](https://docs.google.com/document/d/1IrlcBMrX_TDrRC8U1_QSbEOEJ55pGY7nnCd-nfZfiXM/edit#)
